File: Arbeitsordner_Unity/Assets/Scripts/js/Spieler.js
#pragma strict

var scale : float = 1;
var loose : float = 3;
var gameManager: GameObject;
var gravitationRange : float; // Gravitationsradius

function Start () {
	gravitationRange = GameObject.Find("Settings").GetComponent(Settings).gravitationRange; // Gravitationsradius
	gameManager.GetComponent(CheckApperiance).CheckForScale(scale, gameObject);
	loose = 3;
}

function Update () {
	if (transform.localScale.x > 0) {
		// Kameraposition an Spielerposition anpassen
		Camera.main.transform.position.x = transform.position.x;
		Camera.main.transform.position.y = transform.position.y;	
	}
	
} 

function FixedUpdate () {
	GetComponent(GravitationScript).Gravitation(gravitationRange);
}

function OnCollisionEnter(col: Collision) {
	
	GetComponent(CollisionCheckScript).CollisionCheck(col);
	
}

File: Arbeitsordner_Unity/Assets/Scripts/Spieler.cs
using UnityEngine;
using System.Collections;

public class Spieler : MonoBehaviour {


	public float scale = 1f;
	public float loose;
	public GameObject gameManager;
	public float gravitationRange; // Gravitationsradius

	void  Start (){
		gravitationRange = Settings.settings.gravitationRange; // Gravitationsradius
		// Spielerpositiongenerieren
		transform.position = new Vector3 (0, 0, 0);
	}

	void  Update (){
		if (transform.localScale.x > 0) {
			// Kameraposition an Spielerposition anpassen
			Vector3 updatedPosition = new Vector3(transform.position.x, transform.position.y, -100f);
			Camera.main.transform.position = updatedPosition;
		}
	}
}